asteroid/filterbanks/transforms.py

```diff
@@ -371,17 +385,104 @@ def ebased_vad(mag_spec, th_db: int = 40):
     return log_mag > (max_log_mag - th_db)
```

Comment:
maybe for VAD we should add some simple hangover and hangbefore (easy way is by using median filtering)

Reply:
Would you like to do it? 
Maybe we could simply rely on torchaudio's VAD? I didn't check it but it's probably better than ours, and scriptable.